### PR TITLE
[docs] Add warning on pure_localization submap resolutions

### LINKED ISSUE #847: Different Submaps resolution with pure localization
Can there be any problems / drawbacks running with pure localization on a pbstream which was made with a different submap resolution?

PS: You guys want questions like this here or actually in the `cartographer` repo?
It looked to me like this issue tracker is more for actual interaction with the community than the `cartographer` one. On the other hand it actually is a question on the cartographer core..

### PR DESCRIPTION
**Pull request by AlexReimann**
Closes #847

### CODE REVIEW

**Comment by AlexReimann** [comment]
I signed it!

File: docs/source/tuning.rst [hunk 1]
```diff
@@ -215,6 +215,9 @@ As a next step, we strongly decrease ``global_sampling_ratio`` and ``constraint_
 to compensate for the large number of constraints.
 We then tune for lower latency as explained above until the system reliably works in real time.
 
++If you run in ``pure_localization``, ``submaps.resultion`` **should be matching** with the ``.pbstream`` you are running on.
++Using different solutions are untested and might or might not work.
```
Comment (ojura) [inline docs/source/tuning.rst]: suggestion:
Using different resolutions is currently untested and may not work as expected.

**Comment by AlexReimann** [comment]
Amended using @ojura suggestion and removed `+` which sneaked in by accident

File: docs/source/tuning.rst [hunk 2]
```diff
@@ -215,6 +215,9 @@ As a next step, we strongly decrease ``global_sampling_ratio`` and ``constraint_
 to compensate for the large number of constraints.
 We then tune for lower latency as explained above until the system reliably works in real time.
 
+If you run in ``pure_localization``, ``submaps.resultion`` **should be matching** with the ``.pbstream`` you are running on.
```
Comment (cschuet) [inline docs/source/tuning.rst]: type s/resultion/resolution/g

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 2.

**Pull request merged**